scripts/config.js

```diff
@@ -96,6 +97,14 @@ function getNpmToken () {
   return token;
 }
 
+function getGpgConf () {
+  const gpgPrivateKey = process.env.RPM_GPG_PRIVATE_KEY;
+  const gpgPath = process.env.RPM_GPG_PATH || os.homedir();
+  const gpgName = process.env.RPM_GPG_NAME;
+  const gpgPass = process.env.RPM_GPG_PASS;
+  return { gpgPrivateKey, gpgPath, gpgName, gpgPass}
```

Comment:
Does it pass elint with the missing space before brace?

Reply:
I didn't change the current build process, just amended it to fix the aforementioned missing rpm signature.
No idea about that, though there weren't any eslint complains in the build AFAIR, (probably because those build scripts are not linted ?).
This being said, this is just a convention (I love them, don't misunderstand me) and not a blocker.